core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.directive.html

```diff
@@ -75,50 +198,179 @@ <h2 class="dashboard-title" translate="I18N_DASHBOARD_TOPICS_AND_SKILLS_DASHBOAR
   </div>
 </div>
 <style>
+  .oppia-topics-and-skills-dashboard .intro-card-wrapper {
+    margin: 0 auto;
+    padding-top: 10px;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 0 auto;
+    padding-top: 50px;
+    width: 95%;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-content-primary {
+    background-color: #fff;
+    border: 2px solid #aaaac1;
+    border-radius: 8px;
+    display: inline-block;
+    margin-bottom: 50px;
+    width: 70%;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-content-secondary {
+    display: inline-block;
+    width: 26%;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-list {
+    margin: 30px auto;
+    width: 95%;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-count {
+    align-items: center;
+    background-color: #fff;
+    border: 2px solid #aaaac1;
+    border-radius: 8px;
+    display: flex;
+    flex-wrap: wrap;
+    height: 100px;
+    justify-content: center;
+  }
+  .topic-skill-count .topic-count, .topic-skill-count .skill-count, .create-topic-skill {
+    border-right: 2px solid #aaaac1;
+    display: inline-block;
+    text-align: center;
+    width: 33%;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter {
+    background-color: #fff;
+    border: 2px solid #aaaac1;
+    border-radius: 8px;
+    margin-top: 40px;
+    padding-bottom: 20px;
+  }
+  .oppia-topics-and-skills-dashboard .topic-count-value, .skill-count-value {
+    display: block;
+    font-size: 34px;
+    height: 40px;
+  }
+  .oppia-topics-and-skills-dashboard .create-topic-skill {
+    align-content: center;
+    border-right: none;
+    cursor: pointer;
+    display: flex;
+    flex-wrap: wrap;
+    height: 100%;
+  }
+  .oppia-topics-and-skills-dashboard .create-topic-skill i {
+    color: #00645C;
+    font-size: 30px;
+    width: 100%;
+  }
+  .oppia-topics-and-skills-dashboard .create-topic-skill span {
+    color: #00645C;
+    display: inline-block;
+    font-size: 14px;
+    line-height: 15px;
+    width: 100%;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-content {
+    margin: 0 auto;
+    width: 90%;
+  }
+  .topic-skill-filter-value {
```

Comment:
This is missing the necessary previx. Please double-check all your CSS classes here and elsewhere, there are similar issues in some of the above ones too.

If things get too long, add line breaks after the commas. But it is important to ensure that the CSS classes don't "break out" beyond this directive.

Reply:
Done.